gemma-core/src/main/java/ubic/gemma/core/ontology/OntologyServiceImpl.java

```diff
@@ -803,8 +805,8 @@ private Collection<CharacteristicValueObject> findCharacteristicsFromOntology( S
         List<ubic.basecode.ontology.providers.OntologyService> ontologyServicesToUse;
         if ( useNeuroCartaOntology ) {
             ontologyServicesToUse = Arrays.asList(
-                    nifstdOntologyService,
-                    fmaOntologyService,
+//                    nifstdOntologyService,
+//                    fmaOntologyService,
```

Comment:
Is restricting terms to OBI enough for Neurocarta? We could also get rid of the feature  if it's not too complicated.